### ISSUE TITLE
Add ability to download processed data to a specified folder via a command-line utility

### ISSUE THREAD

**Issue by zachmandeville** [cncf/apisnoop#89]
As an APISnoop developer maintaining the web interface's backend,
who wants a reliable and simple way to bring in new data that does not require a heavy refactor of the backend's existing logic and code, so that I can keep the website current with little time-commitment or room for errors, I want a command-line utility that downloads already processed data to the backend's  specified data folder.

```gherkin
Given a GCS bucket that holds a cache of our processed data,
And a sources.yaml file that outlines the latest builds of this data,
And a web backend that populates its service by consuming files from its local data folder,
When the apisnoop script is run with the flag --download-cache,
Then the latest processed data as outlined ion sources.yaml is downloaded to the backend's data folder,
And this data is formatted to work well with the existing logic of the backend.
```

**Comment by zachmandeville** [comment]
rewriting this for clarity.  What this is referring to is making it so that our web's backend has the means to get new, processed data.  "Pushing" isn't really the way to do that, necessarily.   If we keep all ou8r processed data in a gcs bucket, then the backend could just 'pull' or download from that bucket.  

The spirit of the issue is still valid, but is  lost in the wording, and so I am going to rewrite this entirely.

**Comment by zachmandeville** [comment]
Closed with acfe840d5deb542b97b0329c1a1934a88c3cd379
